packages/cli/src/commands/account/register-data-encryption-key.ts

```diff
@@ -0,0 +1,43 @@
+import { flags } from '@oclif/command'
+import { BaseCommand } from '../../base'
+import { newCheckBuilder } from '../../utils/checks'
+import { displaySendTx } from '../../utils/cli'
+import { Flags } from '../../utils/command'
+
+export default class RegisterDataEncryptionKey extends BaseCommand {
+  static description =
+    'Register a data encryption key for an account where users will be able to retieve the metadata file and verify your claims'
```

Comment:
Command description looks like it needs to be updated. Same goes for the `from` flag. We may want to make sure we mention the fact that the data encryption key will be stored on-chain.